app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,25 @@
+class MessagesController < ApplicationController
+
+  def new
+    @message = Message.new
+  end
+
+  def index
+    @message = Message.all.order created_at: :desc
```

Comment:
remove all and wrap parameters to parentheses

Reply:
what does it mean?